scenariorun/src/main/kotlin/com/cosmotech/scenariorun/workflow/argo/ArgoWorkflowService.kt

```diff
@@ -374,6 +352,57 @@ internal class ArgoWorkflowService(
     return healthBuilder.withDetail("url", apiClient.basePath).build()
   }
 
+  override fun stopWorkflow(scenarioRun: ScenarioRun): ScenarioRunStatus {
+    var workflow = Workflow()
+    try {
+      workflow =
+          newServiceApiInstance<WorkflowServiceApi>(this.apiClient)
+              .workflowServiceStopWorkflow(
```

Comment:
Seems it is not taken into account since there is an issue in the Argo Workflow we use.